.github/workflows/amarna.yml

```
@@ -0,0 +1,26 @@
+name: Amarna Analysis
+on:
+  push:
+    branches:
+      - develop
+      - main
+  pull_request:
+
+jobs:
+  analyze:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run Amarna
+        uses: crytic/amarna-action@v0.1.1
+        id: amarna
+        continue-on-error: true
+        with:
+          sarif: results.sarif
+          target: 'contracts/src/chainlink/cairo/'
+
+      - name: Upload SARIF file
+        uses: github/codeql-action/upload-sarif@v2
+        with:
+          sarif_file: ${{ steps.amarna.outputs.sarif }}
+          checkout_path: ${{ github.workspace }}
```

```suggestion
          checkout_path: '/github/workspace'
```

https://github.com/crytic/amarna-action#how-to-use-1

This should fix incorrect paths/no preview in CodeQL